### PR TITLE
Added GET in cart router

### PR DESCRIPTION
**Pull request by SirChristianWilliam**
check out router

### CODE REVIEW

File: server/modules/routes/cart.routes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router= express.Router();
+const pool = require('../modules/pool');
+
+//GET students (Chris work)
+
+router.get('/', (req,res) => {
```
Comment (AlinaTrukhina) [inline server/modules/routes/cart.routes.js]: router.get does not match filename; propose to change either routes filename or the name in the code

File: server/modules/routes/cart.routes.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    const sqlText = `SELECT * FROM shopping_cart`;
```
Comment (AlinaTrukhina) [inline server/modules/routes/cart.routes.js:8]: could also add line to limit the number of things loaded
i.e. LIMIT 20

File: server/modules/routes/cart.routes.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    pool.query(sqlText)
+    .then((result) => {
+        res.send(result.row);
+    })
+    .catch((err) => {
+        console.log("Error making database query ${sqlText}",err)
```
Comment (AlinaTrukhina) [inline server/modules/routes/cart.routes.js:14]: does this work without variable interpolation?

File: server/modules/routes/cart.routes.js (hunk 1 above)
Comment (SirChristianWilliam) [reply server/modules/routes/cart.routes.js]: Motion passed

File: server/modules/pool.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ const pg = require('pg');
 const Pool = pg.Pool;
 
 const pool = new Pool({
-    database: 'fs-react-shopping', // database name (this will change)
```
Comment (AlinaTrukhina) [inline server/modules/pool.js:8]: database name should be fs-react-shopping.

the table name is shopping_cart.

**Pull request merged**